README.md

```diff
@@ -1,7 +1,72 @@
+# azure-baremetalinfrastructure-cli-extension
+
+# Install
+
+To install this extension just use the CLI extension add command:
+
+```
+az extension add --source $LATEST_RELEASE
+```
+
+# Usage
+
+To list all BareMetal instances for the subscription:
+
+```
+az baremetalinstance list
+```
+
+To show details about a specific BareMetal instance:
+
+```
+az baremetalinstance show --resource-group $RESOURCE_GROUP --instance-name $BM_INSTANCE_NAME
+```
+
+To add a key-value pair to the Tags field of a specific BareMetal instance:
+
+```
+az baremetalinstance update --resource-group $RESOURCE_GROUP --instance-name $BM_INSTANCE_NAME --set tags.newKey=value
+```
+
+To update a key-value pair in the Tags field of a specific BareMetal instance:
+
+```
+az baremetalinstance update --resource-group $RESOURCE_GROUP --instance-name $BM_INSTANCE_NAME --set tags.key=updatedValue
+```
+
+To delete a key-value pair from the Tags field of a specific BareMetal instance:
+
+```
+az baremetalinstance update --resource-group $RESOURCE_GROUP --instance-name $BM_INSTANCE_NAME --remove tags.key
+```
+
+To delete all key-value pairs in the Tags field of a specific BareMetal instance:
+
+```
+az baremetalinstance update --resource-group $RESOURCE_GROUP --instance-name $BM_INSTANCE_NAME --set tags={}
+```
+
+To delete a BareMetal instance:
+
+```
+az baremetalinstance delete --resource-group $RESOURCE_GROUP --instance-name $BM_INSTANCE_NAME
+```
+
+To start a specific BareMetal instance:
+
+```
+az baremetalinstance start --resource-group $RESOURCE_GROUP --instance-name $BM_INSTANCE_NAME
+```
+
+To shutdown a specific BareMetal instance:
```

Comment:
Where is restart?

Reply:
that was an over sight on my part. Will add now